datastore/tests/integration/smoke_test.py

```diff
@@ -137,3 +141,41 @@
 
         after = await ds.runQuery(query, session=s)
         assert len(after.entity_results) == num_results + 3
+
+
+@pytest.mark.skipif('DATASTORE_EXPORT_BUCKET_NAME' not in os.environ,
```

Comment:
Not sure this will work, since the test fixture should throw a `KeyError` if this is unset before it gets to running tests.

Reply:
hmm, pretty sure it did work. it skipped when I didn't have the env var set. I can try again